ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -430,6 +418,8 @@ module.exports = Backbone.AssociatedModel.extend({
       this.repositionLatLon()
     }
 
+    const lat = (north + south) / 2
+    const lon = (east + west) / 2
```

Comment:
❓ how does identifying the middle point of the bounding box help determine if we are in UPS space or not? consider a bounding box spanning from north to south pole, the middle point would be near the equator.

Reply:
These types of checks are used to prevent calls to usng.js that would result in an uncaught error. These errors caused unexpected behavior, like being stuck in drawing mode after the bounding box was drawn instead of going back to the advanced search popup. In this case, usng.js uses the middle point of the bounding box for a call to LLtoUSNG, which in turn calls [getZoneNumber](https://github.com/codice/usng.js/blob/master/src/usng.ts#L231). This function throws an error if the passed coord is in UPS space. Function calls to usng.js were wrapped in a try catch before, but changes were requested to remove them in favor of checking for valid paramters

Reply:
my concern is one or more of your bounding box corner is in UPS space but the middle point of your box is not. Hence you will still be passing coord in the UPS space and cause the function to throws error. So this might not be valid to use the middle point to check, rather it should be check against each corner. If none of the corner is in UPS space u can be sure the whole bounding box isn't as well.

Reply:
The corner's aren't being checked by the getZoneNumber method, the middle point is. That's how this solves to issue.

Reply:
ok due to the nature of this getZoneNumber method. Perhaps better to create a validateUpsZone or something and put this logic in there. So it doesn't look like your doing something incorrect